### DISCUSSION TITLE
Connecting chamber heater to the printer

### DISCUSSION THREAD

**Discussion by Tombraider2006** [Guilouz/Creality-Helper-Script-Wiki#713] [Features Request]
Greetings. wrote a manual on how to connect a heater to the printer and control it with klipper. There are a few nuances, it is only in Russian. And it will be difficult for an inexperienced user to install it, since it will be necessary to edit the config a lot. Perhaps it would be better to integrate it into the installer as one of the menu items? 

[You can read the manual here](https://github.com/Tombraider2006/K1/blob/main/random/heater_chamber/readme.md)

![panel_temp](https://github.com/user-attachments/assets/a7ad3674-68ca-4561-b8ef-4a651ba8a85e)

**Comment by ZloyVovan** [comment]
That would be great, another step towards improving the printer.

**Comment by Vlada3333** [comment]
I agree, good idea!
